Radio/example.jsx

```diff
@@ -99,6 +105,13 @@ export default {
         value='ipsum'
       />,
 
+      'Partially collapsed & controlled': <Radio
+        options={optionsWithContent}
+        expandLabel='Show me all the options'
+        collapsedOptions={2}
```

Comment:
This feels backwards. Wouldn't it make more sense to have this be `visibleOptions` or something like that? If I have a dynamic list of options, I would rather say that I want three of them to be visible than `options.length  - Math.max(options.length - 3, 0)` collapsed.